security/crowdsec/pkg-descr

```diff
@@ -0,0 +1,49 @@
+Crowdsec is an open-source, lightweight software, detecting peers with
+aggressive behaviors to prevent them from accessing your systems. Its user
+friendly design and assistance offers a low technical barrier of entry and
+nevertheless a high security gain.
+
+WWW: https://crowdsec.net/
+
+Plugin Changelog
+================
+
+1.0.0
```

Comment:
Feel free to pick another version for Makefile, I was just concerned about not being able to ship 1.0.0 as the first release if we find issues with it (might look strange)